warehouse/query-core/src/test/java/datawave/query/MaxExpansionRegexQueryTest.java

```diff
@@ -229,6 +236,110 @@ public void testMaxValueAnyFieldNegAnd() throws Exception {
         parsePlan(VALUE_THRESHOLD_JEXL_NODE, 4);
     }
     
+    /**
+     * This test case tests and query that withou an intersection such that when we force the ivarators to fail with a maxSize setting of 1, that the query will
+     * fail.
```

Comment:
"This tests the case of a query without an intersection such that when..."

Reply:
👍